examples/test_data_management.py

```diff
@@ -60,11 +60,16 @@ def _(environment, **_kwargs):
         ).json()["data"]
 
 
-@events.test_stop.add_listener
+@events.test_stopping.add_listener
```

Comment:
Could you update the comment at the start of the file too!

Reply:
like this?
```
# 6. Test run stopping
# 7. User stop
# 8. Test run stop
# 9. (not shown in this example) Locust quit
````